api/v1/trustypb/pkix.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto3";
+package trustypb;
+
+import "rpc.proto";
+import "gogoproto/gogo.proto";
+// for grpc-gateway
+import "google/api/annotations.proto";
+
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service Authority {
+        // ProfileInfo returns the certificate profile info
+        rpc ProfileInfo(CertProfileInfoRequest) returns (CertProfileInfo) {
+            option (google.api.http) = {
+                post: "/v1/ca/csr/profile_info"
+            };
+        }
+
+        // CreateCertificate returns the certificate
+        rpc CreateCertificate(CreateCertificateRequest) returns (CertificateBundle) {
+            option (google.api.http) = {
+                post: "/v1/ca/csr/create"
+            };
+        }
+
+        // CreateCertificate returns the certificate
+        rpc Issuers(EmptyRequest) returns (IssuersInfoResponse) {
+            option (google.api.http) = {
+                get: "/v1/ca/issuers"
+            };
+        }
+}
+
+// X509Name specifies X509 Name
+message X509Name {
+	string country = 1;
+	string state = 2;
+	string locality = 3;
+	string organisation = 4;
+	string organisational_unit = 5;
+}
+
+// X509Subject specifies X509 Subject
+message X509Subject {
+	string cn = 1;
+	repeated X509Name names = 2;
+	string serial_number = 3;
+}
+
+message CertProfileInfoRequest {
+	string label = 1;
+	string profile = 2;
+}
+
+// CertProfileInfo is the response for an Profile Info API request
+message CertProfileInfo {
+	string issuer = 1;
+	repeated string usage = 2;
+	string expiry = 3;
+}
+
+// CertificateBundle provides certificate and its issuers
+message CertificateBundle {
+	// Certificate provides the certificate in PEM format
+	string certificate = 1;
+    // Intermediates provides the intermediate CA certificates bundle in PEM format
+    string intermediates = 2;
+    // Root provides the Root CA certifica in PEM format
+	string root = 3;
```

Comment:
alignment

Reply:
looks like fmt is not applied on `proto` files and spaces are not converted to tabs